velox/exec/tests/utils/OperatorTestBase.cpp

```diff
@@ -54,12 +57,23 @@ OperatorTestBase::~OperatorTestBase() {
 }
 
 void OperatorTestBase::SetUpTestCase() {
-  FLAGS_velox_enable_memory_usage_track_in_default_memory_pool = true;
-  FLAGS_velox_memory_leak_check_enabled = true;
   exec::SharedArbitrator::registerFactory();
   functions::prestosql::registerAllScalarFunctions();
   aggregate::prestosql::registerAllAggregateFunctions();
   TestValue::enable();
+  gAllocator = std::make_unique<memory::MallocAllocator>(kMaxCapacity);
```

Comment:
Would you add comments to gAllocator to explain why we are using global variable (this practice is discouraged)?

Let's move memory manager setup into a helper function to improve readability.